src/components/Toggle.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { View, Switch } from 'react-native';
+import { Icon } from 'react-native-elements';
+
+interface ToggleProps {
+	state: boolean;
+	onChange(val: boolean): void;
+}
+
+const Toggle = ({ state, onChange }: ToggleProps) => {
```

Comment:
I think since this is only being used to switch between enterprise/user modes, we should name this something like "EnterpriseModeToggle" or something more specific, to avoid confusion.

Or if we wanted to make this more a more generic Toggle component, we should allow the icons to be specified as props.